apps/site/src/components/PageViewNFT/index.tsx

```diff
@@ -302,8 +318,9 @@ export default function PageViewNFT({ SSRToken }: Props) {
           width={40}
           height={40}
           alt={token?.name}
-          placeholder="empty"
-          unoptimized={true}
+          placeholder={token.previews?.blurhash ? 'blur' : 'empty'}
+          unoptimized={isPageLoaded}
+          blurDataURL={token.previews?.blurhash}
```

Comment:
This probably won't work as `blurhash` is in a different format. Can you try this? https://www.npmjs.com/package/use-next-blurhash